frontend/src/screens/peers/Peers.tsx

```diff
@@ -94,6 +96,27 @@ function PeerTableRow(props: PeerTableRowProps) {
     return channels?.some((channel) => channel.remotePubkey === peer.nodeId);
   }
 
+  const connectPeer = async (peer: Peer) => {
```

Comment:
We currently don't provide user feedback here, can we add a toast with success/failure message?